### ISSUE TITLE
Improve documentation on enabling SMB support in Linux guest

### ISSUE THREAD

**Issue by jeremyn** [hashicorp/vagrant#7532]
The documentation at Synced Folders > SMB > Common Issues > "wrong fs type" error ([link](https://www.vagrantup.com/docs/synced-folders/smb.html)) says: 'If during mounting on Linux you are seeing an error message that includes the words "wrong fs type," this is because the SMB kernel extension needs to be updated in the OS.' This sounds scary but I got SMB working simply installing the `cifs-utils` package before trying to sync the folder. In other words:

``` ruby
  config.vm.provision "shell", inline: <<-SHELL
      dnf install -y cifs-utils
  SHELL
```

I didn't have to do anything special with the kernel. I did this with Fedora 24 Cloud. I haven't tried it with Ubuntu, but Ubuntu also has a `cifs-utils` package that seems to do the same thing.

Note that because syncing happens before provisioning (see issue #936), I have to first `vagrant up` without smb, and then `vagrant reload` with smb.

Please update the documentation to nudge readers towards the `cifs-utils` package.
### Vagrant version

1.8.4.
### Host operating system

Windows 10.

**Comment by jeremyn** [comment]
It looks like some guest systems have plugins that will automatically install `cifs-utils`, such as Debian at [`plugins/guests/debian/cap/smb.rb`](https://github.com/hashicorp/vagrant/blob/dbb2d992785c7f8cf23bd04f1dbc1e4c1dbe41f4/plugins/guests/debian/cap/smb.rb#L10). So the problem here, arguably, isn't bad documentation but instead that Fedora doesn't have this plugin support. Issue https://github.com/hashicorp/vagrant/issues/7850 is the same problem, but for CentOS.

Could someone at Vagrant please tell me whether we should leave this current issue as it is, or reword it to instead add SMB plugin support for Fedora, or close it as a duplicate of issue https://github.com/hashicorp/vagrant/issues/7850 , or something else?

**Comment by briancain** [comment]
If debian has a capability to install `cifs-utils`, we should be able to do the same for redhat guests. I will close this as a duplicate of #7850. Thanks @jeremyn !
